board/safety/safety_subaru.h

```diff
@@ -20,6 +20,14 @@ const SteeringLimits SUBARU_GEN2_STEERING_LIMITS = {
   .type = TorqueDriverLimited,
 };
 
+const LongitudinalLimits SUBARU_LONG_LIMITS = {
+  .max_brake = 400,  // approx -2.5 m/s^2
+
+  .min_gas = 0,
+  .max_gas = 3400,   // approx  1.2 m/s^2 at low speeds, less at higher speeds
+  .inactive_gas = 808,
+};
+
```

Comment:
```suggestion
const LongitudinalLimits SUBARU_LONG_LIMITS = {
  .max_gas = 3400,   // approx  1.2 m/s^2 at low speeds, less at higher speeds
  .inactive_gas = 808,
  .max_brake = 400,  // approx -2.5 m/s^2
};

```

corrected order. min_gas is only used if the range is not 0 to +x